src/BikeSharing.DomainLogic/BikeWorkout.cs

```diff
@@ -0,0 +1,258 @@
+
+    public class Athlete
+    {
+        private readonly string[] HASHTAGS = new string[]{
+"#transformationtuesday",
+"#mcm",
+"#wcw",
+"#fitfam",
+"#fitspo",
+"#fitness",
+"#gymtime",
+"#treadmill",
+"#gainz",
+"#workout",
+"#getStrong",
+"#getfit",
+"#justdoit",
+"#youcandoit",
+"#bodybuilding",
+"#fitspiration",
+"#cardio",
+"#ripped",
+"#gym",
+"#geekabs",
+"#crossfit",
+"#beachbody",
+"#exercise",
+"#weightraining",
+"#training",
+"#shredded",
+"#abs",
+"#sixpacks",
+"#muscle",
+"#strong",
+"#lift",
+"#weights",
+"#Getfit",
+"#weightloss",
+"#wod",
+"#aesthetic",
+"#squad",
+"#shreadding",
+"#personaltrainer",
+"#dreambitviral",
+"#quote",
+"#quotes",
+"#inspiring",
+"#motivation",
+"#fitnessquote",
+"#youcandoit",
+"#justbringit",
+"#dreambig",
+"#success",
+"#staypositive",
+"#noexcuses"
+    };
+
+        public string Username { get; set; }
+        public Gender Gender { get; set; }
+        public int Age { get; set; }
+        public double Weight { get; set; }
+        public double Height { get; set; }
+        public double BasalMetabolicRate { get; }
+        public List<Workout> Workouts { get; set; }
+
+        public Athlete(string username, Gender gender, int age, double weight, double height)
+        {
+            Username = username;
+            Gender = gender;
+            Age = age;
+            Weight = weight;
+            Height = height;
+            BasalMetabolicRate = GetBasalMetabolicRate();
+        }
+
+        public Athlete(string username, Gender gender, int age, double weight, double height, List<Workout> workouts)
+        {
+            Username = username;
+            Gender = gender;
+            Age = age;
+            Weight = weight;
+            Height = height;
+            BasalMetabolicRate = GetBasalMetabolicRate();
+            Workouts = workouts;
+        }
+
+        public void AddWorkout(params Workout[] w)
+        {
+            if (Workouts != null)
+            {
+                Workouts.AddRange(w);
+            }
+            else
+            {
+                Workouts = w.ToList();
+            }
+        }
+
+        private double GetBasalMetabolicRate()
+        {
+            switch (Gender)
+            {
+                case Gender.Male:
+                    return 66.47 + (13.75 * Weight * 0.453592) + (5.003 * Height * 2.54) - (6.755 * Age);
+                case Gender.Female:
+                    return 655.1 + (9.563 * Weight * 0.453592) + (1.850 * Height * 2.54) - (4.676 * Age);
+                default:
+                    return 0.0;
+            }
+        }
+
+        public double GetCaloriesBurned(Workout workout)
+        {
+            //var a = (Gender)5;
+            switch (Gender)
+            {
+                case Gender.Male:
+                    return (-55.0969 + (0.6309 * workout.AverageHeartRate) 
+                           + (0.1988 * Weight) + (0.2017 * Age) / 4.184) 
+                           * 60 * workout.Duration.TotalHours;
```

Comment:
I would add a function that makes this more obvious.  There are a lot of magic numbers here